GraphQL Docs for Parse Server 3.10.0 (#688)
ParsePlatform/Docs: _includes/graphql/users.md

To sign up a new user, use the `signUp` mutation. For example:

```graphql
mutation signUp {
  signUp(
    input: {
      userFields: {
        username: "johndoe"
        password: "ASuperStrongPassword"
        email: "john.doe@email.com"
      }
    }
  ) {
    viewer {
      sessionToken
      user {
        username
        email
      }
    }
  }
}

> 

Reviewer: ```suggestion
```

You can log out a user through the `logOut` mutation. You need to send the `X-Parse-Session-Token` header and run code like the below example:

```js
// Header
{
  "X-Parse-Application-Id": "APPLICATION_ID",
  "X-Parse-Session-Token": "r:b0dfad1eeafa4425d9508f1c0a15c3fa"
}
```
```graphql
# GraphQL
mutation logOut {
  logOut(input: { clientMutationId: "logOut" }) {
    clientMutationId
    viewer {
      user {
        username
        email
      }
    }
  }
}

```
```js
// Response
{
  "data": {
    "logOut": {
      "clientMutationId": "logOut",
      "viewer": {
        "user": {
          "username": "johndoe",
          "email": "john.doe@email.com"
        }
      }
    }
  }
}
```

> ## Reset Password

Reviewer: In line with other guides...
```suggestion
## Resetting Passwords
```

> Currently reset password is not supported, but this will be addressed soon.

Reviewer: ```suggestion
Currently resetting passwords is not supported, but this will be addressed soon.
```

> Currently reset password is not supported a pull request should be sent soon.

Reviewer: ```suggestion
Currently reset password is not supported, but this will be addressed soon.
```
